src/connection/arrayconnection.js

```diff
@@ -153,7 +153,7 @@ export function cursorForObjectInConnection<T>(
  * otherwise it will be the default.
  */
 export function getOffsetWithDefault(
-  cursor?: ?ConnectionCursor,
+  cursor?: ConnectionCursor | null,
```

Comment:
technically it can't be optional since second argument is required
```suggestion
  cursor: ConnectionCursor | null,
```

Reply:
Scratch it, looks like 
```suggestion
  cursor: ConnectionCursor | undefined,
```
is what we want based on real usage scenarios

Reply:
`cursor` comes from resolver args, so `ConnectionCursor | undefined | null` would make sense